custom_metrics/almanac.js

```diff
@@ -307,5 +307,15 @@ return JSON.stringify({
+    // words = privacy_words.map(country => country.words);
+    // pattern = words.flat().join('|');
+
+    var pattern = 'privatësia|politika e të dhënave|konfidencialiteti|politikat e privatësisë|politikat e privatesise|mbrojtja e të dhënave|datenschutz|datenrichtlinie|privacy|gegevensbeleid|confidentialité|politique d’utilisation des données|privacybeleid|условия за ползване|поверителност|политика за данни|политика за бисквитки|условия|privacidade|política de dados|privacy|data policy|confidentialité|politique d’utilisation des données|datenschutz|datenrichtlinie|privacy|data policy|confidentialité|politique d’utilisation des données|隐私权政策|数据使用政策|私隱政策|數據使用政策|gizlilik|veri i̇lkesi|απόρρητο|πολιτική δεδομένων|soukromí|zásady používání dat|zásady používání dat|ochrana soukromí|podmínky|ochrana dat|ochrana osobních údajů|datenschutz|datenrichtlinie|privatssphäre|datenschutzbestimmungen|datenschutzrichtlinie|beskyttelse af personlige oplysninger|datapolitik|cookiepolitik|privatlivspolitik|personoplysninger|regler om fortrolighed|personlige data|privaatsus|data policy|kasutustingimused|isikuandmete|isikuandmete töötlemise|küpsised|konfidentsiaalsuse|andmekaitsetingimused|الخصوصية|سياسة البيانات|privacidad|política de datos|protecció de dades|aviso legal|yksityisyys|tietokäytäntö|tietosuojakäytäntö|yksityisyyden suoja|tietosuojaseloste|rekisteriseloste|tietosuoja|yksityisyydensuoja|confidentialité|politique d’utilisation des données|mentions légales|cgu|cookies|vie privée|donnees personelles|mentions légales|cgv|απόρρητο|όροι και γνωστοποιήσεις|προσωπικά δεδομένα|πολιτική απορρήτου|privatnost|privatnosti|pravila o upotrebi podataka|zaštita podataka|kolačići|adatvédelem|adatkezelési|adatvédelmi|személyes adatok védelme|privacy|data policy|privacy policy|cookie policy|פרטיות|מדיניות נתונים|persónuvernd|privacy|normativa sui dati|privatumas|slapukai|privatumo|confidentialité|politique d’utilisation des données|datenschutz|datenrichtlinie|privātums|privātuma|sīkdatņu|sīkdatne|приватност|политика о подацима|Privatnost|приватност|политика о подацима|privatezza|privacidad|política de datos|gegevensbeleid|privacybeleid|personvern|retningslinjer for data|prywatność|zasady dotyczące danych|prywatności|privacidade|política de dados|confidențialitate|politica de utilizare|cookie-uri|confidentialitate|cookie-urilor|protecţia datelor|приватност|политика о подацима|конфиденциальность|политика использования данных|политика конфиденциальности|политика обработки персональных данных|sekretess|datapolicy|personuppgifter|webbplatsen|integritetspolicy|zasebnost|piškotkih|varstvo podatkov|ochrana súkromia|zásady využívania údajov|ochrana údajov|ochrana osobných údajov|súkromie|piškotki|zásady ochrany osobných|osobné údaje|ochrany osobných údajov|gizlilik|veri i̇lkesi|veri politikası|kvkk|kişisel verilerin korunması|конфіденційність';
+
+    return Array.from(document.querySelectorAll('a')).filter(a => a.innerText.match(new RegExp(pattern, 'i'))).length;
```

Comment:
Create the RegExp object once rather than for every `a`.
```suggestion
    var pattern = new RegExp('privatësia|politika e të dhënave|konfidencialiteti|politikat e privatësisë|politikat e privatesise|mbrojtja e të dhënave|datenschutz|datenrichtlinie|privacy|gegevensbeleid|confidentialité|politique d’utilisation des données|privacybeleid|условия за ползване|поверителност|политика за данни|политика за бисквитки|условия|privacidade|política de dados|privacy|data policy|confidentialité|politique d’utilisation des données|datenschutz|datenrichtlinie|privacy|data policy|confidentialité|politique d’utilisation des données|隐私权政策|数据使用政策|私隱政策|數據使用政策|gizlilik|veri i̇lkesi|απόρρητο|πολιτική δεδομένων|soukromí|zásady používání dat|zásady používání dat|ochrana soukromí|podmínky|ochrana dat|ochrana osobních údajů|datenschutz|datenrichtlinie|privatssphäre|datenschutzbestimmungen|datenschutzrichtlinie|beskyttelse af personlige oplysninger|datapolitik|cookiepolitik|privatlivspolitik|personoplysninger|regler om fortrolighed|personlige data|privaatsus|data policy|kasutustingimused|isikuandmete|isikuandmete töötlemise|küpsised|konfidentsiaalsuse|andmekaitsetingimused|الخصوصية|سياسة البيانات|privacidad|política de datos|protecció de dades|aviso legal|yksityisyys|tietokäytäntö|tietosuojakäytäntö|yksityisyyden suoja|tietosuojaseloste|rekisteriseloste|tietosuoja|yksityisyydensuoja|confidentialité|politique d’utilisation des données|mentions légales|cgu|cookies|vie privée|donnees personelles|mentions légales|cgv|απόρρητο|όροι και γνωστοποιήσεις|προσωπικά δεδομένα|πολιτική απορρήτου|privatnost|privatnosti|pravila o upotrebi podataka|zaštita podataka|kolačići|adatvédelem|adatkezelési|adatvédelmi|személyes adatok védelme|privacy|data policy|privacy policy|cookie policy|פרטיות|מדיניות נתונים|persónuvernd|privacy|normativa sui dati|privatumas|slapukai|privatumo|confidentialité|politique d’utilisation des données|datenschutz|datenrichtlinie|privātums|privātuma|sīkdatņu|sīkdatne|приватност|политика о подацима|Privatnost|приватност|политика о подацима|privatezza|privacidad|política de datos|gegevensbeleid|privacybeleid|personvern|retningslinjer for data|prywatność|zasady dotyczące danych|prywatności|privacidade|política de dados|confidențialitate|politica de utilizare|cookie-uri|confidentialitate|cookie-urilor|protecţia datelor|приватност|политика о подацима|конфиденциальность|политика использования данных|политика конфиденциальности|политика обработки персональных данных|sekretess|datapolicy|personuppgifter|webbplatsen|integritetspolicy|zasebnost|piškotkih|varstvo podatkov|ochrana súkromia|zásady využívania údajov|ochrana údajov|ochrana osobných údajov|súkromie|piškotki|zásady ochrany osobných|osobné údaje|ochrany osobných údajov|gizlilik|veri i̇lkesi|veri politikası|kvkk|kişisel verilerin korunması|конфіденційність', 'i');

    return Array.from(document.querySelectorAll('a')).filter(a => a.innerText.match(pattern)).length;
```